ajax/libs/bootstrap-year-calendar/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "bootstrap-year-calendar",
+  "description": "A fully customizable year calendar widget, for Bootstrap!",
+  "homepage": "http://www.bootstrap-year-calendar.com/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Paul-DS/bootstrap-year-calendar"
+  },
+  "author": "Paul-DS <@Paul-DS> (https://taylorhakes.com/)",
+  "license": "Apache-2.0",
+  "filename": "bootstrap-year-calendar.js",
```

Comment:
`filename` should point to the main minified file, `js/bootstrap-year-calendar.min.js`.